Bio/motifs/jaspar/db.py

```diff
@@ -92,8 +91,7 @@ class JASPAR5(object):
     """
 
     def __init__(self, host=None, name=None, user=None, password=None):
-        """
-        Construct a JASPAR5 instance and connect to specified DB
+        """Construct a JASPAR5 instance and connect to specified DB
```

Comment:
You could insert the missing final full stop (period) while you are working on this.